### PR TITLE
Add note about numbers with four or more digits needing commas

### PR DESCRIPTION
**Pull request by vr8hub**
As discussed on the list.

### CODE REVIEW

**Comment by acabal** [comment]
This should probably be its own rule because it does not relate to the question of spelling vs. digits. Can you do that?

**Comment by vr8hub** [comment]
Sure.

**Comment by vr8hub** [comment]
I added it as the next rule after the existing one, and added the shortest wrong/corrected example I could find.

File: 8-typography.rst [hunk 1]
```diff
@@ -760,7 +760,7 @@ Numbers, measurements, and math
 
 			<p>The 1st, 2nd, 3rd, 4th.</p>
 
-#.	Numbers in a non-mathematical context are spelled out if they are less than or equal to 100. Numbers over 100 are set with digits.
```
Comment (acabal) [inline 8-typography.rst]: Let's keep this though, for clarity

Comment (vr8hub) [reply 8-typography.rst]: Good grief, sorry about that. I deleted my added sentence and apparently also picked up the previous one by mistake. Fixed.

**Comment by acabal** [comment]
Thanks!

**Pull request merged**